dingo/volestipy.pyx

```diff
@@ -18,7 +18,6 @@ import os
 import sys
 import numpy as np
 cimport numpy as np
-from libcpp cimport bool
```

Comment:
are we sure this is not needed @hariszaf @TolisChal ?

Reply:
i think it's not needed. we struggled a lot with the boolean vars and most probably it's the following line
```
from cpython cimport bool
```
that does the work.  

if the tests do not fail,  i guess it's fine. apologies for not having a clearer answer.